static/js/containers/ProfileImage_test.js

```diff
@@ -1,26 +1,29 @@
 /* global SETTINGS: false */
-import '../global_init';
-
 import React from 'react';
 import { mount } from 'enzyme';
 import { assert } from 'chai';
 import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
 import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import { Provider } from 'react-redux';
 
 import ProfileImage from './ProfileImage';
 import IntegrationTestHelper from '../util/integration_test_helper';
+import ProfileImageUploader from '../components/ProfileImageUploader';
+import * as api from '../lib/api';
+import { startPhotoEdit, updatePhotoEdit } from '../actions/image_upload';
 
 describe('ProfileImage', () => {
-  let helper;
+  let helper, sandbox, updateProfileImageStub;
 
-  const renderProfileImage = (profile, editable=true) => (
+  const renderProfileImage = (props, editable=true) => (
     mount(
       <MuiThemeProvider muiTheme={getMuiTheme()}>
-        <ProfileImage
-          profile={profile} 
-          store={helper.store}
-          editable={editable}
-        />
+        <Provider store={helper.store}>
+          <ProfileImage
+            editable={editable}
+            {...props}
+          />
+      </Provider>
```

Comment:
the JSX indentation for https://github.com/pangloss/vim-javascript is a bit screwy sometimes...